src/components/Button/CozyHomeLink.jsx

```diff
@@ -2,29 +2,27 @@ import React from 'react'
 import PropTypes from 'prop-types'
 import { ButtonLink } from 'cozy-ui/react'
 import styles from './index.styl'
+import getHomeLinkHref from './getHomeLinkHref'
 
-const CozyHomeLink = ({ from, embedInCozyBar = false, t }) => (
+const CozyHomeLink = ({ from, embedInCozyBar = false, t, size, className }) => (
   <ButtonLink
     label={t('Share.create-cozy')}
     icon="cozy-negative"
-    className={embedInCozyBar ? styles['bar-homelink'] : ''}
+    className={embedInCozyBar ? styles['bar-homelink'] : className}
     href={getHomeLinkHref(from)}
+    size={size}
   />
 )
 
 CozyHomeLink.propTypes = {
   from: PropTypes.string,
   embedInCozyBar: PropTypes.bool,
-  t: PropTypes.func.isRequired
+  t: PropTypes.func.isRequired,
+  size: PropTypes.oneOf(['tiny', 'small', 'large'])
```

Comment:
`size` has no default value and in one of the cases, we don't provide it as a prop, so it will be undefined and probably produce proptypes warning. Could you make sure there are no warnings coming from this code please?